CommunityToolShedMvc/CommunityToolShedMvc/Controllers/ToolsController.cs

```diff
@@ -0,0 +1,97 @@
+﻿using CommunityToolShedMvc.Data;
+using CommunityToolShedMvc.Models;
+using CommunityToolShedMvc.Security;
+using CommunityToolShedMvc.ViewModels;
+using System;
+using System.Collections.Generic;
+using System.Data.SqlClient;
+using System.Linq;
+using System.Web;
+using System.Web.Mvc;
+
+namespace CommunityToolShedMvc.Controllers
+{
+    [Authorize]
+    public class ToolsController : Controller
+    {
+
+
+        // GET: Tools
+
+
+        public ActionResult Detail(int communityid, int id)
```

Comment:
Looks like the "Detail" page is about borrowing tools. I'd rename this action method and the associated POST action method to something like "Borrow".